content/flutter-configuration/built-in-variables.md

```diff
@@ -1,6 +1,7 @@
 ---
-description: Useful variables exported during builds
-title: Built-in environment variables
+description: Useful variables exported during builds using Flutter Workflow Editor
```

Comment:
Do we capitalize all first letters in Flutter Workflow Editor or do we capitalize on the F in Flutter?

Reply:
Good point. I don't know for sure. Statistically, we use 'Flutter workflow editor' more often but it could be that I introduced that statistic with recent docs rewrite. Our main page (https://docs.codemagic.io/) has 'Flutter workflow editor" above but then also "Flutter Workflow Editor' immediately below if you scroll just a bit.

I am ok with both so please choose one :-D

Reply:
Looked at the blog and it's the same. Let it be `Flutter workflow editor` similarly to how it's `Codemagic` rather than `CodeMagic` :)